modules/cluster-config-operator.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// *  operators/operator-reference.adoc
+
+[id="cluster-config-operator_{context}"]
+= Cluster Config Operator
+
+[discrete]
+== Purpose
+
+The Cluster Config Operator performs the following tasks related to `config.openshift.io`:
+
+. Creates CRDs.
```

Comment:
These should be bullets and not a numbered list.

Reply:
Thanks for reviewing. Addressed.